src/components/logo/styles.tsx

```diff
@@ -2,8 +2,6 @@ import styled from 'styled-components'
 
 import { Heading2 } from 'styles/typography/styles'
 
-const Heading = styled(Heading2)`
+export const Heading = styled(Heading2)`
   color: ${({ theme }) => theme.colors.white};
 `
```

Comment:
Could you put a prop color or type in Heading2 for setting color without creation wrapper?